api/v1alpha1/hypershiftdeployment_types.go

```diff
@@ -80,16 +80,24 @@ type HypershiftDeploymentSpec struct {
 	InfraID string `json:"infra-id,omitempty"`
 
 	// InfrastructureOverride allows support for special cases
-	//   InfraOverrideDestroy = "ORPHAN"
+	//   OverrideDestroy = "ORPHAN"
 	//   InfraConfigureOnly = "INFRA-ONLY"
-	//   InfraConfigureWithManifest = "MANIFESTWORK"
+	//   ConfigureWithManifest = "MANIFESTWORK"
 	// +kubebuilder:validation:Enum=ORPHAN;INFRA-ONLY;MANIFESTWORK
 	Override InfraOverride `json:"override,omitempty"`
 
-	//TargetNamespace allows the children resource(like, manifestwork) to be
-	//generated in other than the hypershiftDpeloyment's namespace.
+	//TargetNamespace specify the where the children resouces(hostedcluster, nodepool)
+	//to sit in other than the hypershiftDeployment's namespace
+	// +optional
 	TargetNamespace string `json:"targetNamespace"`
 
+	//TargetManagedCluster specify which managedcluster's namespace the manifestwork would be deployed.
+	//If not specified, it uses the hypershiftDeployment.
+	//The managedcluster would be the management cluster of the hostedcluster and nodepool generated
```

Comment:
Replace `The managedcluster` with `The TargetManagedCluster`  This is a little clearer.